mmv1/third_party/terraform/acctest/bootstrap_test_utils.go

```diff
@@ -358,9 +358,13 @@ func BootstrapSharedTestNetwork(t *testing.T, testId string) string {
 
 const SharedTestGlobalAddressPrefix = "tf-bootstrap-addr-"
 
-func BootstrapSharedTestGlobalAddress(t *testing.T, testId, networkId string) string {
+// params := [prefixLength] prefixLength should be the first item in params
```

Comment:
Might be able to use something like an `*Options` struct so that this is more explicit

Reply:
I will test it later with *Options struct. I wonder if all of function calls should change to add this parameter.

Reply:
Either options structs or functional options (https://www.sohamkamani.com/golang/options-pattern/) are preferable to variadic arguments. Structs make more sense for us for the most part- they're cheaper, mainly. Functional options are often used in the interface to public libraries.

Reply:
I am going to use the functional options, as it will not affect most existing function calls.